cmake/modules/FindWaylandpp.cmake

```diff
@@ -55,14 +54,19 @@ find_package_handle_standard_args(Waylandpp
                                   VERSION_VAR WAYLANDPP_wayland-client++_VERSION)
 
 if(WAYLANDPP_FOUND)
+  if(WAYLANDPP_STATIC_LDFLAGS)
+    set(PC_WAYLANDPP_STATIC_LDFLAGS ${WAYLANDPP_STATIC_LDFLAGS} CACHE INTERNAL "" FORCE)
```

Comment:
Might want to set this the other way around as I don't think manually setting pkg-config variables is a nice thing to do.